__tests__/query.test.js

```diff
@@ -116,12 +116,13 @@ const SCHOOLDETAILS_DATA = [
 const CREDENTIALS_DATA = [
   {
     id: '1',
-    name: 'Masters in Gravitational Engineering',
+    credName: 'Masters in Gravitational Engineering',
```

Comment:
Great change. Semantics matter. We'll need to refactor FE now.

Reply:
Yeah, had that in mind and figured that the earlier field name changes would be done, the better